templates/golem.service.j2

```diff
@@ -5,7 +5,12 @@ After=network.target hyperdrive.service
 [Service]
 Type=simple
 User={{ golem_source_user }}
-ExecStart={{ golem_source_path }}/venv/bin/python {{ golem_source_path }}/golemapp.py -d {{ golem_source_config_dir }} {{ golem_source_unit_extra_args }}
+ExecStart={{ golem_source_path }}/venv/bin/python {{ golem_source_path }}/golemapp.py \
+    -d {{ golem_source_config_dir }} \
+    {{ golem_source_mainnet | ternary('--mainnet', '') }} \
+    --accept-terms \
+    --password {{ golem_source_password }} \
```

Comment:
I think this should be in double quotes just in case

Reply:
done